extensions/exposition/features/identity.federation.feature

```diff
@@ -54,6 +54,34 @@ Feature: Identity Federation
       id: ${{ User.id }}
       """
 
+  Scenario: Getting identity for a user with symmetric tokens
+    Given the `identity.federation` configuration:
+      """yaml
+      explicit_identity_creation: false
+      trust:
+        - issuer: http://localhost:44444
+          secret: the-secret
+      """
+    And the IDP symmetric token for GoodUser is issued with following secret:
```

Comment:
`the IDP token for GoodUser is issued using `HS256` symmetric algorithm`

Name of the algorithm must be a parameter.